src/Language/PureScript/Sugar/Operators.hs

```diff
@@ -1,22 +1,3 @@
------------------------------------------------------------------------------
---
--- Module      :  Language.PureScript.Sugar.Operators
--- Copyright   :  (c) 2013-15 Phil Freeman, (c) 2014-15 Gary Burgess
--- License     :  MIT (http://opensource.org/licenses/MIT)
---
--- Maintainer  :  Phil Freeman <paf31@cantab.net>
--- Stability   :  experimental
--- Portability :
---
--- |
--- This module implements the desugaring pass which reapplies binary operators based
```

Comment:
Oops, yep. I did so with the other ones I took the header out of but missed it here.